.github/workflows/ci.yml

```diff
@@ -23,10 +23,21 @@ jobs:
           submodules: true
           persist-credentials: false
 
-      - name: Install Deno
-        uses: denolib/setup-deno@master
-        with:
-          deno-version: ${{ matrix.deno-version }}
+      - name: Install Deno (Unix)
+        if: |
+          !startsWith(matrix.os, 'windows')
+        run: |-
+          curl -fsSL https://deno.land/x/install/install.sh | sh
+          echo "$HOME/.deno/bin" >> $GITHUB_PATH
+      - name: Install Deno (Windows)
+        if: startsWith(matrix.os, 'windows')
+        run: |-
+          curl -fsSL https://deno.land/x/install/install.sh | sh
+          echo "$HOME/.deno/bin" >> $env:GITHUB_PATH
```

Comment:
Can't we just set `bash` as the default shell globally instead of doing all this branching?

Reply:
Doesn't work. We tried on denoland/deno repo already.